src/library/baseexternallibraryfeature.cpp

```diff
@@ -33,6 +34,13 @@ BaseExternalLibraryFeature::~BaseExternalLibraryFeature() {
     delete m_pImportAsMixxxPlaylistAction;
 }
 
+void BaseExternalLibraryFeature::bindSidebarWidget(WLibrarySidebar* pSidebarWidget) {
+    // store the sidebar widget pointer for later use in onRightClickChild
+    m_pSidebarWidget = pSidebarWidget;
+    qDebug() << "";
+    qDebug() << "   BaseExternalLibraryFeature::bindSidebarWidget("<< pSidebarWidget <<")";
```

Comment:
this debug should be removed ore commented.

Reply:
this slipped through.. fixed.